linux-packages/openziti-console/nfpm-openziti-console.yaml

```diff
@@ -0,0 +1,16 @@
+# nfpm configuration file
+#
+# check https://nfpm.goreleaser.com/configuration for detailed usage
+#
+name: openziti-console
+arch: ${GOARCH}
+platform: linux
+version: ${ZAC_VERSION}
+maintainer: ${ZITI_MAINTAINER}
+description: Web console static files for OpenZiti Controller
```

Comment:
how about: "A single-page web application for managing an OpenZiti overlay network"